### DISCUSSION TITLE
Why is there a need of installing powershell

### DISCUSSION THREAD

**Discussion by jobe39** [MartijnvanGeffen13/Volvo4evcc#29] [Q&A] [answered]
Wanted to try this approach, since it takes care about a lot of details, other moduls don´t (and does not need an mqtt broker), but than I saw, that it requires powershell to be installed. What is the reason behind it, to use powershell and not any other linux based shell scripting?

Thanks for explanation.

br,
  Gernot

**Comment by MartijnvanGeffen13** [comment, marked as answer]
i am a PowerShell Master Trainer :) 

I know PowerShell, and it works cross version windows  / *ux / Mac
